src/model/cloud-runner/aws/aws-task-runner.ts

```diff
@@ -0,0 +1,216 @@
+    } catch (error_) {
+      const error = error_ as Error;
+      await new Promise((resolve) => setTimeout(resolve, 3000));
+      CloudRunnerLogger.log(
+        `Cloud runner job has ended ${
+          (await AWSTaskRunner.describeTasks(ECS, cluster, taskArn)).containers?.[0].lastStatus
+        }`,
+      );
+
+      core.setFailed(error);
+      core.error(error);
+    }
+    CloudRunnerLogger.log(`Cloud runner job is running`);
+    await this.streamLogsUntilTaskStops(ECS, CF, taskDef, cluster, taskArn, streamName);
+    const exitCode = (await AWSTaskRunner.describeTasks(ECS, cluster, taskArn)).containers?.[0].exitCode;
+    CloudRunnerLogger.log(`Cloud runner job exit code ${exitCode}`);
+    if (exitCode !== 0 && exitCode !== undefined) {
+      core.error(
+        `job failed with exit code ${exitCode} ${JSON.stringify(
+          await ECS.describeTasks({ tasks: [taskArn], cluster }).promise(),
+          undefined,
+          4,
+        )}`,
+      );
+      throw new Error(`job failed with exit code ${exitCode}`);
+    } else {
+      CloudRunnerLogger.log(`Cloud runner job has finished successfully`);
+    }
+  }
+
+  static async describeTasks(ECS: AWS.ECS, clusterName: string, taskArn: string) {
+    const tasks = await ECS.describeTasks({
+      cluster: clusterName,
+      tasks: [taskArn],
+    }).promise();
+    if (tasks.tasks?.[0]) {
+      return tasks.tasks?.[0];
+    } else {
+      throw new Error('No task found');
+    }
+  }
+
+  static async streamLogsUntilTaskStops(
+    ECS: AWS.ECS,
+    CF: AWS.CloudFormation,
+    taskDef: CloudRunnerTaskDef,
+    clusterName: string,
+    taskArn: string,
+    kinesisStreamName: string,
+  ) {
+    const kinesis = new AWS.Kinesis();
+    const stream = await AWSTaskRunner.getLogStream(kinesis, kinesisStreamName);
+    let iterator = await AWSTaskRunner.getLogIterator(kinesis, stream);
+
+    CloudRunnerLogger.log(
+      `Cloud runner job status is ${(await AWSTaskRunner.describeTasks(ECS, clusterName, taskArn))?.lastStatus}`,
+    );
+
+    const logBaseUrl = `https://${Input.region}.console.aws.amazon.com/cloudwatch/home?region=${AWS.config.region}#logsV2:log-groups/log-group/${taskDef.taskDefStackName}`;
+    CloudRunnerLogger.log(`You can also see the logs at AWS Cloud Watch: ${logBaseUrl}`);
+    let readingLogs = true;
+    let timestamp: number = 0;
+    while (readingLogs) {
+      await new Promise((resolve) => setTimeout(resolve, 1500));
+      const taskData = await AWSTaskRunner.describeTasks(ECS, clusterName, taskArn);
+      ({ timestamp, readingLogs } = AWSTaskRunner.checkStreamingShouldContinue(taskData, timestamp, readingLogs));
+      ({ iterator, readingLogs } = await AWSTaskRunner.handleLogStreamIteration(
+        kinesis,
+        iterator,
+        readingLogs,
+        taskDef,
+      ));
+    }
+  }
+
+  private static async handleLogStreamIteration(
+    kinesis: AWS.Kinesis,
+    iterator: string,
+    readingLogs: boolean,
```

Comment:
Prefer prefixing booleans where possible using common convention.

- `is`
- `has`
( or `should` ? not sure)

i.e. `shouldReadLogs` or `isReadingEnabled` or `isReaderEnabled`

Reply:
Will fix